rs/migration_canister/src/lib.rs

```diff
@@ -30,7 +30,12 @@ mod processing;
 mod tests;
 mod validation;
 
-const DEFAULT_MAX_ACTIVE_REQUESTS: u64 = 50;
+/// The max number of requests in a 24 hour sliding window. Requests are either
+/// - active (in REQUESTS)
+/// - succeeded (in HISTORY) and not older than 24 hours.
+const RATE_LIMIT: u64 = 50;
+/// Validations cause xnet calls, so we limit them.
+const MAX_ONGOING_VALIDATIONS: u64 = 500;
```

Comment:
This limit should be reduced since the current value can saturate the queue to the management canister and cause some of the ongoing validated requests to fail. We should have the maximum number of validated ongoing requests + the maximum number of requests being validated + some slack <= 500 (queue capacity).